connectors/src/api/unpause_connector.ts

```diff
@@ -37,9 +37,11 @@ const _unpauseConnectorAPIHandler = async (
       );
       return res.sendStatus(204);
     }
-    const connectorUnpauser = UNPAUSE_CONNECTOR_BY_TYPE[connector.type];
 
-    const unpauseRes = await connectorUnpauser(connector.id);
+    const unpauseRes = await getConnectorManager({
```

Comment:
Can we leverage the `connectorResource`? Since most methods fetch it afterward, if we already have it (which we usually do), we should pass it to the ConnectorManager.

Reply:
Sure we can do that in a later iteration (not pushing back, but I want to keep this PR very simple and focused on just moving the function implementations into class methods)